iesg_statements/2007-03-20-0.md

```diff
@@ -188,7 +188,7 @@ Date: Mar 2007
 
    AD Sponsored submissions represent a significant workload to the IESG.  Reasons for the popularity of these submissions include the interest of the ADs to progress work in their fields, the difference in time-to-RFC-publication IETF documents enjoy over independent submissions, the ability to avoid the IESG notes that independent submissions get, and the wider review IETF documents get.
 
-   Improvements in the efficiency of the RFC Editor processing are likely to increase the popularity of the independent submissions, which represent a smaller load for the IESG.  Similarly, ongoing work [I-D.klensin-rfc-independent](#8-references) may change the tone of the IESG notes.  However, the speed of the independent submissions channel depends to a large extent on its review stage, and it has generally been easier to find reviewers for IETF documents.
+   Improvements in the efficiency of the RFC Editor processing are likely to increase the popularity of the independent submissions, which represent a smaller load for the IESG.  Similarly, ongoing work I-D.klensin-rfc-independent may change the tone of the IESG notes.  However, the speed of the independent submissions channel depends to a large extent on its review stage, and it has generally been easier to find reviewers for IETF documents.
```

Comment:
same here on getting the brackets back

Reply:
Here, this is a footnote style reference, not a word in the sentence, so it would need the '[]'